.travis.yml

```diff
@@ -21,6 +33,8 @@ jobs:
 #        - java -jar target/yasson-jmh.jar -t 1 -f 2
     - stage: checkstyle
       script: mvn checkstyle:checkstyle
+    - stage: copyright
```

Comment:
is is necessary to have a separate stage that runs the copyright check? If I'm reading the pom correctly, we are binding the copyright check goal to the `validate` phase which will run as part of the first stage that does a `mvn clean install`

Reply:
oh... did I not remove that binding to validate phase? Sorry I might have left it there. If so I will definitely remove it.

Reply:
well, I think the binding to a phase is good, because it's less extra steps that get run by CI (running the TCK and checkstyle plugins are good exceptions though).

How exactly does this copyright plugin work? Does it automatically add/update the copyright for us? Or does it simply fail when it finds something out of place?

Reply:
I took a look at the settings and this plugin is not running by default, you have to allow it by profile as it is also in case of checkstyle. And that is the reason why I added it also to pipeline, because we just want to be sure. I also do not think that adding it to the basic build is the way to go, because it will bother you even when some of the new files are just temporary.

There are 2 goals which I want to highlight: check and repair
check -> checks files in repository if they have proper copyright present there and updated year. (if not excluded) It will fail and prints out the files and reason why it failed.
repair -> adds and updates copyright to every file which has been flagged as incorrect.